Release/include/cpprest/http_client.h

```diff
@@ -347,6 +348,25 @@ class http_client_config
     {
         return m_ssl_context_callback;
     }
+
+    /// <summary>
+    /// Gets the TLS server name indication (SNI) property.
+    /// </summary>
+    /// <returns>True if TLS server name indication is enabled, false otherwise.</returns>
+    bool tlsext_host_name() const
```

Comment:
Thanks for the feedback. I was unsure whether you want to deviate from the current naming scheme in the API which does not phrase "bool-getters" as an "is_something..." question, will make the adjustments shortly.